docs/source/terms_and_model.rst

```diff
@@ -346,7 +346,119 @@ Location for Variation.
+chromosomal band within a species.  Conceptually, a ChromosomeLocation
+represents a family of :ref:`SequenceLocation` on distinct
+:ref:`Sequence` entities.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Location Id; MUST be unique within document
+   * - type
+     - string
+     - 1..1
+     - Location type; MUST be set to **'ChromosomeLocation'**
+   * - species
+     - :ref:`CURIE`
+     - 1..1
+     - An external reference to a species taxonomy.  See Implementation Guidance, below.
+   * - chromsome
+     - string
+     - 1..1
+     - The symbolic chromosome name
+   * - start
+     - string
+     - 1..1
+     - The chromosmal band name of the start
+   * - end
+     - string
+     - 1..1
+     - The end chromosomal band name; if null, end is presumed to equal `start`.
+
+
+**Implementation guidance**
+
+* ChromosomeLocation is intended to enable the representation of
+  cytogenetic results from karyotyping or low-resolution molecular
+  methods.  Precise :ref:`SequenceLocation` should be preferred when
+  nucleotide-scale location is known.
+* `species` is specified using the NCBI taxonomy.  The CURIE prefix
+  MUST be `taxonomy`, corresponding to the `NCBI taxonomy prefix at
+  identifiers.org
+  <https://registry.identifiers.org/registry/taxonomy>`__.
+* Valid values for, and syntactic structure of, `chromosome`, `start`,
+  and `end` depend on the species.  Therefore, VRS does not constrain
+  these values in the model, except for the guidelines below.
+* `chromosome` MUST be an official primary name from `NCBI Assembly
+  <https://www.ncbi.nlm.nih.gov/assembly>`__.  For Humans, valid
+  chromosome names are 1..22, X, Y.
+* `start` and `end` SHOULD be values that are conventional for the
+  species. For Humans, bands are denoted by the arm (`p` or `q`) and
+  position (e.g., `22` or `22.3`). See example.
+* Prescribing the conversion of ChromosomeLocations to
+  SequenceLocations is out-of-scope for VRS.  Recommended data for
+  this operation are available at `NCBI GDP
+  <https://ftp.ncbi.nlm.nih.gov/pub/gdp/>`__, `UCSC GRCh37 (hg19)
+  <http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/cytoBand.txt.gz>`__,
+  `UCSC GRCh38 (hg38)
+  <http://hgdownload.cse.ucsc.edu/goldenPath/hg38/database/cytoBand.txt.gz>`__,
+  and `bioutils (Python)
+  <https://bioutils.readthedocs.io/en/stable/reference/bioutils.cytobands.html>`__.
+* Design decision: The tuple <species,chromosome name> is used to
+  refer an archetypal chromosome for the species.  WikiData and MESH
+  provide similar definitions (e.g., Human Chr 1 at `WikiData
+  <https://www.wikidata.org/wiki/Q430258>`__ and `MESH
+  <https://meshb.nlm.nih.gov/record/ui?ui=D002878>`__, but don't
+  provide sufficient benefit to merit the added complexity in VRS at
+  this time.
```

Comment:
I think we had decided that, if suitable concepts were found, that it _would_ be worth the complexity, but that the issue was about the stability / governance of WikiData concepts (and sparsity of MESH concepts). This design decision should reflect that, and perhaps be added directly in the _Design Decisions_ section instead of the definition.

Reply:
I don't think we agreed that it would be worth the complexity (or at least I didn't), but you're right that we discussed data stability in a crowd-sourced entity and overall coverage in MESH. I'll add it.